src/msw/listctrl.cpp

```diff
@@ -1780,37 +1774,41 @@ wxListCtrl::HitTest(const wxPoint& point, int& flags, long *ptrSubItem) const
 
     flags = 0;
 
-    if ( hitTestInfo.flags & LVHT_ABOVE )
-        flags |= wxLIST_HITTEST_ABOVE;
-    if ( hitTestInfo.flags & LVHT_BELOW )
-        flags |= wxLIST_HITTEST_BELOW;
+    // test whether an actual item was hit or not
+    if ( hitTestInfo.flags == LVHT_ONITEMICON )
+        flags = wxLIST_HITTEST_ONITEMICON;
+    // note a bug in comtl32.dll:
+    // 1) in report mode, when there are 2 or more columns, if you click to the
+    //    right of the first column, ListView_HitTest() returns LVHT_ONITEM
+    //    (LVHT_ONITEMICON|LVHT_ONITEMLABEL|LVHT_ONITEMSTATEICON).
+    else if ( hitTestInfo.flags == LVHT_ONITEMLABEL ||
```

Comment:
Couldn't/shouldn't we just test for `flags & LVHT_ONITEMLABEL`? I.e. are there any cases when `LVHT_ONITEMLABEL` is set, but we don't want to return `wxLIST_HITTEST_ONITEMLABEL`?